val-ansible-role/valkey/tasks/main.yml

```diff
@@ -0,0 +1,51 @@
+---
+- name: Include OS-specific variables
+  include_vars: "{{ ansible_os_family }}.yml"
+  ignore_errors: yes
+
+- name: Ensure valkey user exists
+  user:
```

Comment:
use fcqn
 ansible.builtin.user
 refer below for details
 https://ansible.readthedocs.io/projects/lint/rules/fqcn/